aws_lambda_powertools/utilities/connections/redis.py

```diff
@@ -0,0 +1,133 @@
+                    db=self.db_index,
+                    decode_responses=True,
+                    **self.extra_options,
+                )
+        except redis.exceptions.ConnectionError as exc:
+            logger.debug(f"Cannot connect in Redis: {self.host}")
+            raise RedisConnectionError("Could not to connect to Redis", exc) from exc
+
+        return self._connection
+
+
+class RedisStandalone(RedisConnection):
+    def __init__(
+        self,
+        host: Optional[str] = None,
+        port: Optional[int] = None,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        db_index: Optional[int] = None,
+        url: Optional[str] = None,
+        **extra_options,
+    ) -> None:
+        """
+        Initialize the Redis standalone client
+        Parameters
+        ----------
+        host: str
+            Name of the host to connect to Redis instance/cluster
+        port: int
+            Number of the port to connect to Redis instance/cluster
+        username: str
+            Name of the username to connect to Redis instance/cluster in case of using ACL
+            See: https://redis.io/docs/management/security/acl/
+        password: str
+            Passwod to connect to Redis instance/cluster
+        db_index: int
+            Index of Redis database
+            See: https://redis.io/commands/select/
+        url: str
+            Redis client object configured from the given URL
+            See: https://redis.readthedocs.io/en/latest/connections.html#redis.Redis.from_url
+        """
+        super().__init__(redis.Redis, host, port, username, password, db_index, url, **extra_options)
+
+
+class RedisCluster(RedisConnection):
+    def __init__(
+        self,
+        host: Optional[str] = None,
+        port: Optional[int] = None,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        db_index: Optional[int] = None,
+        url: Optional[str] = None,
+        **extra_options,
+    ) -> None:
+        """
+        Initialize the Redis standalone client
+        Parameters
+        ----------
+        host: str
+            Name of the host to connect to Redis instance/cluster
+        port: int
+            Number of the port to connect to Redis instance/cluster
+        username: str
+            Name of the username to connect to Redis instance/cluster in case of using ACL
+            See: https://redis.io/docs/management/security/acl/
+        password: str
+            Passwod to connect to Redis instance/cluster
+        db_index: int
+            Index of Redis database
+            See: https://redis.io/commands/select/
+        url: str
+            Redis client object configured from the given URL
+            See: https://redis.readthedocs.io/en/latest/connections.html#redis.Redis.from_url
+        """
+
+        super().__init__(redis.cluster.RedisCluster, host, port, username, password, db_index, url, **extra_options)
```

Comment:
If I understand correctly, the only difference between the two classes is the first parameter (`redis.cluster.RedisCluster` vs `redis.Redis`).

It that's the only difference, wouldn't it make sense to have a single class with a flag for `standalone` vs `cluster` instead of 2 different classes?